### DISCUSSION TITLE
Power On Create3

### DISCUSSION THREAD

**Discussion by YXGuan** [iRobotEducation/create3_docs#323] [Questions] [answered]
To power on create 3, the only way is to " plug your charging dock in and place the robot on the dock" ? 

Would it be possible to turn the robot on without the charging dock?

**Comment by justinIRBT** [comment, marked as answer]
When the robot is fully powered off, the dock is the only way to turn it on.

There is a “standby” mode as described in the button 1 description here https://iroboteducation.github.io/create3_docs/hw/face/ that will turn off power to a subset of the robot and can be woken up from a button press, but note that the robot will continue to drain some battery while in that mode, so it cannot be used over a long period of time.

**Reply by YXGuan** [reply]
Sounds good. Thank you for your response Justin!
